### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Pacoscript**

### CODE REVIEW

File: staff/francisco-fernandez/cv-styled/styles.css [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    display: grid;
+    grid-template-columns: repeat(12,1fr);
+    grid-template-rows: auto;
+    padding: 10px;
+    width: auto;
+  }
+
+  /* For mobile phones: */
+    [class*="item"] {
+        grid-column-start: 2;
```
Comment (seand52) [inline staff/francisco-fernandez/cv-styled/styles.css]: This is good, but you might want to consider start using the shorthand: 
grid column: start / end 
that way you can have less code

File: staff/francisco-fernandez/cv-styled/index.html [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" type="text/css" href="styles.css">
+    <title>Curriculum vitae de Patricia Medina</title>
+</head>
+
+<body>
+    <header>
+        <h1>CV Patricia Medina López</h1>
+    </header>
+    <section class="grid-container">
```
Comment (vior1992) [inline staff/francisco-fernandez/cv-styled/index.html:18]: Maybe you can use "MAIN" for englobe this part?

File: staff/francisco-fernandez/cv-styled/index.html [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+        <div class="item1 leftcontainer">
```
Comment (vior1992) [inline staff/francisco-fernandez/cv-styled/index.html:19]: better use "__" for BEM methodology?

File: staff/francisco-fernandez/cv-styled/styles.css [hunk 4]
```diff
@@ -0,0 +1,115 @@
+header{
```
Comment (diesttro) [inline staff/francisco-fernandez/cv-styled/styles.css:5]: You need some general styles for the page like font-family, font-size, color, etc...

File: staff/francisco-fernandez/cv-styled/styles.css (hunk 1 above)
Comment (Pacoscript) [reply staff/francisco-fernandez/cv-styled/styles.css]: I've changed it, yes it´s better, less code

File: staff/francisco-fernandez/cv-styled/index.html (hunk 2 above)
Comment (Pacoscript) [reply staff/francisco-fernandez/cv-styled/index.html:18]: Yes, I think you are right, I did it

File: staff/francisco-fernandez/cv-styled/index.html (hunk 3 above)
Comment (Pacoscript) [reply staff/francisco-fernandez/cv-styled/index.html:19]: Yes but it was not my intention to use BEM, i used two classes in the div. Its not a good way of work, i will change it in the future

File: staff/francisco-fernandez/cv-styled/styles.css (hunk 4 above)
Comment (Pacoscript) [reply staff/francisco-fernandez/cv-styled/styles.css:5]: Yes, you are right... I´ve done

**Pull request merged**